docs/blog/posts/qdrant-to-pgvector.md

```diff
@@ -0,0 +1,30 @@
+---
+authors:
+  - Carlos
+category:
+  - Database
+  - Architecture
+date: 2024-03-19
+---
+# Ditching Qdrant for PgVector
+
+
+In our latest infrastructure update, we decided to transition from Qdrant to pgvector for managing our vector databases. This move is part of our ongoing effort to reduce cost and simplify AAQ’s architecture.
+
+
+<!-- more -->
+
+This means:
+- we no longer require a separate Qdrant server. With the pgvector extension, vector data can now be stored in PostgreSQL along with other transactional data.
+- Seamless transactions between vector data and transactional data will allow for the integration of new features such as multilingual support.
+
+
+## Why this change?
+
+While Qdrant served us well as a dedicated vector database, integrating it with our existing PostgreSQL setup introduced complexity and maintenance overhead. Operating Qdrant alongside PostgreSQL meant managing two distinct database systems with their own infrastructure, architecture, and requirements. Integrating both databases into our codebase required additional integration layers, complicating our codebase.
+
+As we already were using Postgresql, pgvector caught our attention as a promising solution that integrates vector database capabilities directly into our existing database. Here's why we decided to go for pgvector:
```

Comment:
Can you link those blogs Zia found that compare performance and show that pg_vector with HSNW is pretty decent? That's what convinced us that it was time to move.